<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projeto Conjunto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(240, 240, 240);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Conjunto A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Conjunto B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Tamanho :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Incluir Elemento :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TamanhoConjuntoA">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_InserirElementoConjuntoA">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_TamanhoConjuntoB">
    <property name="geometry">
     <rect>
      <x>328</x>
      <y>50</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_InserirElementoConjuntoB">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>90</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>90</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Incluir Elemento :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>50</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Tamanho :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConjuntoA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Conjunto A :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConjuntoB">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>130</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="text">
     <string>Conjunto B :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TamanhoConjuntoA">
    <property name="geometry">
     <rect>
      <x>151</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_InserirElementoConjuntoA">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_InserirElementoConjuntoB">
    <property name="geometry">
     <rect>
      <x>453</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Inserir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TamanhoConjuntoB">
    <property name="geometry">
     <rect>
      <x>413</x>
      <y>50</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>Criar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Uniao">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>170</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>A U B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Intersecao">
    <property name="geometry">
     <rect>
      <x>142</x>
      <y>170</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>A âˆ© B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Diferenca_AB">
    <property name="geometry">
     <rect>
      <x>272</x>
      <y>170</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>A - B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Diferenca_BA">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>170</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>B - A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>250</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>RESULTADO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_SaidaResultado">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>280</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MaiorSubCadeia_Crescente">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>210</y>
      <width>132</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>&gt; SubCadeia Crescente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_MaiorSubCadeia_Decrescente">
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>210</y>
      <width>132</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>&gt; SubCadeia Decrescente</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_Mais">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>351</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>O conjunto e vazio?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>E subconjunto do outro?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SÃ£o identifcos?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>SÃ£o disjuntos?</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Amplitude</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Produto Escalar</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>MÃ©dia Aritmetica</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>CONJUNTO A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CONJUNTO B</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_Mais">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS PGothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>â†“</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Menos">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>â†‘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UniaoOrdenada">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>132</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);</string>
    </property>
    <property name="text">
     <string>A U B (Ordenada)</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>150</y>
      <width>490</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>10</y>
      <width>10</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>335</y>
      <width>490</width>
      <height>12</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>240</y>
      <width>490</width>
      <height>10</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>pushButton_Menos</zorder>
   <zorder>pushButton_Mais</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit_TamanhoConjuntoA</zorder>
   <zorder>lineEdit_InserirElementoConjuntoA</zorder>
   <zorder>lineEdit_TamanhoConjuntoB</zorder>
   <zorder>lineEdit_InserirElementoConjuntoB</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_ConjuntoA</zorder>
   <zorder>label_ConjuntoB</zorder>
   <zorder>pushButton_TamanhoConjuntoA</zorder>
   <zorder>pushButton_InserirElementoConjuntoA</zorder>
   <zorder>pushButton_InserirElementoConjuntoB</zorder>
   <zorder>pushButton_TamanhoConjuntoB</zorder>
   <zorder>pushButton_Uniao</zorder>
   <zorder>pushButton_Intersecao</zorder>
   <zorder>pushButton_Diferenca_AB</zorder>
   <zorder>pushButton_Diferenca_BA</zorder>
   <zorder>label_7</zorder>
   <zorder>label_SaidaResultado</zorder>
   <zorder>pushButton_MaiorSubCadeia_Crescente</zorder>
   <zorder>pushButton_MaiorSubCadeia_Decrescente</zorder>
   <zorder>tableWidget_Mais</zorder>
   <zorder>pushButton_UniaoOrdenada</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
   <zorder>line_3</zorder>
   <zorder>line_4</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
